src/EA.Weee.RequestHandlers/Admin/Obligations/GetPcsObligationsCsvHandler.cs

```diff
@@ -56,7 +56,7 @@ public async Task<CSVFileData> HandleAsync(GetPcsObligationsCsv message)
 
             var fileContent = csvWriter.Write(authority.Schemes);
             var timestamp = DateTime.Now;
-            var fileName = $"{authority.Abbreviation}_pcsobligationuploadtemplate{timestamp.ToString("dd/MM/yyyy_HHmm")}.csv";
+            var fileName = $"{authority.Abbreviation}_pcsobligationuploadtemplate{timestamp.ToString("_yyyy_dd/MM/yyyy_HHmm")}.csv";
```

Comment:
Can we write a test for this....

Reply:
Yes, although the infrastructure for the request handler unit tests is still sitting in my unit tests branch, should I add it there?

Reply:
Sounds good